packages/core/elasticsearch/core-elasticsearch-server-internal/src/get_cluster_info.ts

```diff
@@ -28,6 +29,7 @@ export function getClusterInfo$(internalClient: ElasticsearchClient): Observable
       cluster_name: info.cluster_name,
       cluster_uuid: info.cluster_uuid,
       cluster_version: info.version.number,
+      cluster_build_flavor: info.version.build_flavor,
```

Comment:
As said previously, I reused the `client.info` call from this method to build the capabilities, which surfaced the `cluster_build_flavor` in a few files / snapshots.  Overall, I don't think this is a problem so I didn't bother filtering / excluding it in some places.